wandb/sdk/wandb_run.py

```diff
@@ -1854,77 +1888,118 @@ def save(
             )
             return True
 
-        return self._save(glob_str, base_path, policy)
+        if isinstance(glob_str, bytes):
+            # Preserved for backward compatibility: allow bytes inputs.
+            glob_str = glob_str.decode("utf-8")
+        elif isinstance(glob_str, str) and (
+            glob_str.startswith("gs://") or glob_str.startswith("s3://")
```

Comment:
hmmm... this is probably a trivial test, should we add? (i think we had some request to actually support remote files, would be kinda nice to do it... :) )